src/test/scala/com/datastax/gatling/plugin/simulations/cql/BoundCqlTypesSimulation.scala

```diff
@@ -44,7 +47,10 @@ class BoundCqlTypesSimulation extends BaseCqlSimulation {
        |:tinyint_type, :time_type, :null_type, :udt_type, :tuple_type, :frozen_set_type, :set_string_type
        |)""".stripMargin
 
-  val preparedStatementSelect = s"""SELECT * FROM $testKeyspace.$table_name WHERE uuid_type = ?"""
+  val preparedStatementSelect = QueryBuilder.selectFrom(testKeyspace, table_name)
+    .all()
+    .whereColumn("uuid_type").isEqualTo(QueryBuilder.bindMarker())
+    .build()
```

Comment:
I didn't make a systematic effort to convert all the old inline CQL into schemabuilder calls.  This could be done as a future exercise... it certainly seems like it's more readable.